website/docs/reference/resource-configs/databricks-configs.md

```diff
@@ -427,11 +427,11 @@ To take advantage of this capability, you will need to add compute blocks to you
 
         ### Name that you will use to refer to an alternate compute
        Compute1:
-          http_path: [‘/sql/your/http/path’] # Required of each alternate compute
+          http_path: ['/sql/your/http/path'] # Required of each alternate compute
 
         ### A third named compute, use whatever name you like
         Compute2:
-          http_path: [‘/some/other/path’] # Required of each alternate compute
```

Comment:
Why are square brackets needed?  There should be just a single `http_path` per compute entry, not a list, correct?

Reply:
you're right @johnson-jay-l ! the brackets were previously used to display code/fields that needed to be replaced with the user content. however, it can cause confusion for sure. I've removed them so it's clearer - thanks for flagging!